src/GraphQL.Federation/GraphQL.Federation.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.1;netstandard2.0</TargetFrameworks>
```

Comment:
Any `netstandard2.1` specific APIs here?

Reply:
Not. removed 2.1

Reply:
Without netstandard2.1, I am not able to build harness project, was getting "GraphQL.Federation is not compatible" error.

Reply:
This is strange. Show an error.